examples/reminders/lib/base/services/reminders_service.dart

```diff
@@ -10,6 +11,8 @@ class RemindersService {
 
   final RemindersRepository _repository;
 
+  static const _nameValidation = 'A title must be specified';
```

Comment:
It might be, if we were in the UI layer or we were using system of error models and translating them depending on the model. But here in the service we don have context to access the localizations.